pkg/apply/apply.go

```diff
@@ -74,12 +68,10 @@ func ApplyObject(ctx context.Context, client k8sclient.Client, obj *uns.Unstruct
 	err := client.Get(ctx, types.NamespacedName{Name: obj.GetName(), Namespace: obj.GetNamespace()}, existing)
 
 	if err != nil && apierrors.IsNotFound(err) {
-		log.Printf("does not exist, creating %s", objDesc)
```

Comment:
In my opinion, These logs seem useful in debugging issues. Shouldn't we try to keep some of them around? Perhaps we can construct a string and pass it back to the caller to log.

Reply:
I believe we mainly care if Apply succeeded or not (and why it failed in case of error)
the rest is noise. current logs are already pretty verbose.